tsconfig.json

```diff
@@ -40,6 +40,16 @@
     "forceConsistentCasingInFileNames": true,
     "rootDir": "."
   },
+  "ts-node": {
+    "experimentalSpecifierResolution": "node",
+    "transpileOnly": true,
+    "esm": true,
+    "swc": true,
+    "compilerOptions": {
+      "jsxFactory": "qwikJsx.h",
+      "jsxFragmentFactory": "qwikJsx.Fragment"
+    }
```

Comment:
Can you try adding https://www.npmjs.com/package/ts-node#require here so that the runBefore is always executed?